lib/web_ui/lib/src/engine/skwasm/skwasm_impl/renderer.dart

```diff
@@ -399,7 +399,8 @@ class SkwasmRenderer implements Renderer {
   }
 
   @override
-  Future<void> renderScene(ui.Scene scene) => sceneView.renderScene(scene as EngineScene);
+  Future<void> renderScene(ui.Scene scene, ui.FlutterView view) =>
+      sceneView.renderScene(scene as EngineScene);
```

Comment:
Can you please file an issue and attach a TODO here? Skwasm should implement multi-view without too much delay.